docs/pages/config/projects/upstreams.mdx

```diff
@@ -55,10 +55,16 @@ projects:
           minBudget: 0
           maxBudget: 10_000
 
-        # chainId is optional and will be detected from the endpoint (eth_chainId),
-        # but it is recommended to set it explicitly, for faster initialization.
         evm:
-          chainId: 42161]
+          # chainId is optional and will be detected from the endpoint (eth_chainId),
+          # but it is recommended to set it explicitly, for faster initialization.
+          chainId: 42161
+          # statePollerInterval is required and can be used to change the interval for checking the node's liveness. Default is 30s.
+          statePollerInterval: 5s
+          # nodeType is optional and you can manually set it to "full" or "archive"
+          nodeType: full
+          # maxAvailableRecentBlocks is optional and can be used to limit the maximum number of recent blocks to be served by this upstream. Default is 128 for full nodes.
+          maxAvailableRecentBlocks: 128
```

Comment:
was thinking maybe to capitalize optional
```suggestion
          # (OPTIONAL) chainId is optional and will be detected from the endpoint (eth_chainId),
          # but it is recommended to set it explicitly, for faster initialization.
          chainId: 42161
          # (OPTIONAL) statePollerInterval used to periodically fetch the latest/finalized/sync states. DEFAULT: 30s.
          statePollerInterval: 30s
          # (OPTIONAL) nodeType is optional and you can manually set it to "full" or "archive". DEFAULT: archive
          nodeType: full
          # (OPTIONAL) maxAvailableRecentBlocks limits the maximum number of recent blocks to be served by this upstream. DEFAULT: 128 (for "full" nodes).
          maxAvailableRecentBlocks: 128
```

Reply:
I thought based on this (https://github.com/erpc/erpc/blob/9fbb437d58e8aaca8c934a32f89ce130540d56b5/common/validation.go#L369), `statePollerInterval` is required?